src/app/Login/page.tsx

```diff
@@ -0,0 +1,91 @@
+"use client"
+import { useState } from "react"
+import { Container, Typography, TextField, Button, Snackbar } from "@mui/material"
+import { useNavigate } from "react-router-dom"
+import './Login.css'
+
+function Login() {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [error, setError] = useState<string | null>(null)
+    const [openSnackbar, setOpenSnackbar] = useState<boolean>(false)
+    //const navigate = useNavigate()
+
+    const handleLogin = () => {
+        fetch('../api/mock/user/login/route', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({ email, password }),
+        })
+            .then((response) => {
+                if (response.ok) {
+                    return response.json()
+                } else {
+                    throw new Error('Login failed')
+                }
+            })
+            .then((data) => {
+                if (data.success) {
+                    //navigate('/')
+                    // Uncaught Error: useNavigate() may be used only in the context of a <Router> component.
+                    setError("success")
+                    setOpenSnackbar(true)
+                } else {
+                    setError(data.error_message)
+                    setOpenSnackbar(true)
+                }
+            })
+            .catch((error) => {
+                console.error('error: ', error)
+            })
+
+
+    }
+
+    return (
+        <Container className="container" maxWidth="sm">
+            <div className="form">
+                <Typography variant="h4" component="h2" gutterBottom>
+                    Login
+                </Typography>
+                <form>
```

Comment:
```suggestion
                <form onSubmit={(e)=> {
                  e.preventDeafult();
                  handleLogin();
                }}>
```
어차피 form 쓰셨으니 submit 이벤트로 처리하는 것도 좋아 보입니다! `preventDefault`하는 거 잊지 마시구요~